modules/ztp-support-for-deployment-of-multi-node-clusters.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: CONCEPT
+[id="ztp-support-for-deployment-of-multi-node-clusters_{context}"]
+= ZTP support for deployment of multi-node clusters
+
+The Telco 5G zero touch provisioning (ZTP) flow uses the Assisted Service, which is part of {rh-rhacm-first} on the hub cluster, to install clusters. This is done by generating all of the custom resources (CRs) required by Assisted Service including:
+
+* `AgentClusterInstall`
+* `ClusterDeployment`
+* `NMStateConfig`
+* `ManagedCluster and `KlusterletAddonConfig` (integration with {rh-rhacm})
+* `InfraEnv`
+* `BareMetalHost`
+* `ConfigMap` for extra install manifests
+
+Extending ZTP to support three-node clusters and standard clusters requires updates to these CRs,including multiple instantiations of some.
+
+ZTP provides support for deploying single node clusters, three-node clusters, and standard OpenShift clusters. This includes the installation of OpenShift and deployment of the distributed units (DUs) at scale.
+
+The overall flow is identical to the ZTP support for single node clusters, with some differentiation in configuration depending on the type of cluster:
+
+`SiteConfig`:
+
+* For single node clusters, the `SiteConfig` file must have exactly one entry in the `nodes` section.
+* For three-node clusters, the `SiteConfig` file must have exactly three entries defined in the `nodes` section.
+* For standard clusters, the `SiteConfig` file must have exactly three entries in the `nodes` section with `role: master` and one or more additional entries with `role: worker`.
```

Comment:
For standard clusters this should read: "two or more additional entries with role:worker"